content/knowledge_base/flow/cadence/resources.md

```diff
@@ -0,0 +1,96 @@
+# Resources
+
+Resources are types that can exist only in one location at a time.
+At the end of a function which has resources in scope, resources must either be **moved** or **destroyed**.
```

Comment:
So what "location" does mean in that context?

Reply:
For me personally I think it's okay as it is, since it's described in more detail below. But I was also reading about resources beforehand, so a fresher perspective is more valuable :)

Reply:
Yeah.